networking_cisco/plugins/cisco/cfg_agent/device_drivers/asr1k/aci_asr1k_routing_driver.py

```diff
@@ -462,6 +462,19 @@ def _asr_do_remove_floating_ip(self, floating_ip,
         self._edit_running_config(confstr,
                                   'REMOVE_STATIC_SRC_TRL_NO_VRF_MATCH')
 
+    def _snat_prefix(self, subnet):
+        snat_id = NAT_POOL_PREFIX + subnet['id'][:self.NAT_POOL_ID_LEN]
+        if cfg.CONF.multi_region.enable_multi_region:
+            prefix = "%s-%s" % (snat_id, cfg.CONF.multi_region.region_id)
+        else:
+            prefix = snat_id
+        return prefix
+
+    def _get_snat_prefix(self, ri, port, ext_port):
```

Comment:
I added this in case we needed the port object in order to be able to support other mappings. I can remove this if you prefer, and we can add it in later if we find that we need it.